### PR TITLE
Add support for content library info modules

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>
See: https://github.com/ansible-collections/vmware.vmware_rest/issues/149

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
refresh_modules.py

### CODE REVIEW

**Comment by goneri** [comment]
We also need to add functional tests for these new modules in https://github.com/ansible-collections/vmware.vmware_rest

**Comment by Akasurde** [comment]
@goneri Could you please review and merge this? Thanks in advance.

File: vmware_rest_code_generator/module_utils/vmware_rest.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import importlib
 from ansible.module_utils.basic import missing_required_lib
 from ansible.module_utils.parsing.convert_bool import boolean
+from ansible.module_utils.six import string_types
```
Comment (goneri) [inline vmware_rest_code_generator/module_utils/vmware_rest.py:5]: Please don't use six. The module only works with py3.6+

**Review by goneri — CHANGES_REQUESTED** [review]
Thank you Abhijeet, I will land this PR as soon as the six import thing is resolved.

**Pull request merged**